README.md

```diff
@@ -2,6 +2,7 @@
 
 Running:
 ```shell
-cargo run --release -- -d ../gir-files Gtk-3.0
+cargo run --release -- -c Gir_Gtk.toml -d ../gir-files -o ./gtk
```

Comment:
Just `gtk` might be nicer looking than `./gtk`, also maybe just call it something neutral like `out_dir`?